.gitignore

```diff
@@ -7,4 +7,5 @@ out/
 /jni/generated/
 /jni/cmake-build-*/
 /jni/venv
-*.log
\ No newline at end of file
+*.log
+prebuilt
```

Comment:
this is outside the build directory, so it isn't removed by gradle clean

Reply:
but why shouldn't it be deleted by gradle clean?

Reply:
Because they are not part of the build for which you run gradle clean, they come from somewhere else (ie. a remote mac)